app/assets/javascripts/merged_plate.js

```diff
@@ -0,0 +1,28 @@
+(function($, exports, undefined){
+  'use strict'
+
+  //= require lib/keycodes
+
+  $(function(_event) {
+    // If we are not on the multi-plate pooling page, don't set anything up.
+    if ($('#merged-plate-page').length === 0) { return }
+
+    // Build up a list of all fields we may want to tab through.
+    var fields = $('#new_plate').find("input[type=text],input[type=submit]")
```

Comment:
Where does this new_plate id come from then?

Reply:
The rails form helper automatically sets the form id to new_plate